modules/openid_connect/app/models/openid_connect/provider.rb

```diff
@@ -144,7 +144,7 @@ def icon
     end
 
     def scopes
-      (scope || "").split
+      (scope || "openid email profile").split.map(&:to_sym)
```

Comment:
Soft question: I would have expected this to be a list of strings. Though that's only a gut feeling, if there are reasons to choose sym or string, feel free to point them out.

I agree that the list should not be a mix of both.